CI-test/Foo.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Foo.swift
+//  CI-test
+//
+//  Created by Eugen Spinu on 10/01/2021.
+//
+
+import Foundation
+
+struct Foo {
+    let param1: Int
+    let param2: Int
+    
+    func foo() -> Int {
+        _ = foo1()
+        return param1+param2
+    }
+    func foo1() -> Int {
```

Comment:
<table data-meta="generated_by_metrics">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/espinu/iOS-CI-test/blob/ac1c2af2959fc51c14dac281b09d9589ea2f795a/CI-test/Foo.swift#L18"></span>Function should have complexity 5 or less: currently complexity equals 12<br />
<code>cyclomatic_complexity</code> <code>Foo.swift:18</code></td>
    </tr>
  </tbody>
</table>